src/scripts/main.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const blockEl = document.createElement('div');
+  const titleEl = document.createElement('h2');
+  const textEl = document.createElement('p');
+
+  blockEl.classList = `notification ${type}`;
+  titleEl.classList = 'title';
```

Comment:
Similar to the previous comment, use `className` instead of `classList` to assign a string of classes.